server/models/User.coffee

```diff
@@ -264,12 +264,7 @@ UserSchema.statics.unconflictName = unconflictName = (name, done) ->
 
 UserSchema.methods.register = (done) ->
   @set('anonymous', false)
-  if (name = @get 'name')? and name isnt ''
-    unconflictName name, (err, uniqueName) =>
-      return done err if err
-      @set 'name', uniqueName
-      done()
-  else done()
+  done()
```

Comment:
Would removing `done` mean that it waits for the sendwithus stuff to happen? I'm not sure how it interacts with those parts when you call `done()` before doing other stuff.